src/error/__tests__/GraphQLError-test.ts

```diff
@@ -12,6 +10,7 @@ import { parse } from '../../language/parser';
 import { Source } from '../../language/source';
 
 import { GraphQLError, printError } from '../GraphQLError';
+import { OperationDefinitionNode, ObjectTypeDefinitionNode } from '../../index';
```

Comment:
Don't use `index` imports.
Also since we don't distribute our tests it's perfectly fine to use TS 3.9 features in it so please use a new TS syntax for importing types.

Reply:
Sure, fixed. will push in a minute.